libdnf/dnf-sack.cpp

```diff
@@ -226,7 +226,14 @@ dnf_sack_new(void)
 static int
 current_rpmdb_checksum(Pool *pool, unsigned char csout[CHKSUM_BYTES])
 {
-    const char *rpmdb_prefix_paths[] = { "/var/lib/rpm/Packages",
```

Comment:
We can drop  support of DNF_SACK_LOAD_FLAG_BUILD_CACHE for system repo.
There are two good reasons for that:
1. The calculation of checksumm doesn't work correctly in some cases (the reason why DNF stop to support it)
2. There is no performance benefit

Reply:
Heh, discovery and dropping of non-working baggage is one of the better possible outcomes :grin:

Reply:
Please don't forget we'll need similar functionality in the new dnf-daemon we're going to deliver as part of dnf5 work.
Basically something to help the daemon recognize that rpmdb has changed and it should reload data.